src/formBuilder/CardGeneralParameterInputs.js

```diff
@@ -243,6 +249,27 @@ export default function CardGeneralParameterInputs({
           />
         </div>
       </div>
+      <div className='card-entry-row'>
```

Comment:
I wonder if the Modal would be a more fitting place for thisl.

Reply:
Seems like it, what are you recommendation in achieving this as I found it hard to add input to the default model without affecting fields that might not require this field.

Reply:
This is a good point. We may potentially need to add it to the `modalBody` of each of the relevant `defaultInputs` individually then?

Reply:
Hey Raymond, I've updated the PR to include the above changes in the relevant modalBody's. This also includes creating references custom model.